strict/src/Data/Strict/Tuple.hs

```diff
@@ -163,21 +162,6 @@ unzip x = ( map fst x
 -- Instances
 ------------
 
-#if __GLASGOW_HASKELL__ >= 608
-deriving instance (Data a, Data b) => Data (Pair a b)
-#endif
-#if MIN_VERSION_base(4,7,0)
-deriving instance Typeable Pair
-#else
-deriving instance Typeable2 Pair
```

Comment:
You may wonder where this goes, but GHC says to use `deriving (Typeable)` when deriving clause is attached to type definition. `typeOf2` (with old GHC / base) still work, i.e. there is `Typeable2` instance still.

Reply:
Is this just for backwards-compat? I thought deriving Typeable was implicit these days.